packages/gafl-webapp-service/src/pages/order-complete/__tests__/order-complete.spec.js

```diff
@@ -41,11 +41,9 @@ describe('The order completion handler', () => {
   it('throw a status 403 (forbidden) exception if the finalized flag is not set', async () => {
     await JUNIOR_12_MONTH_LICENCE.setup()
 
-    salesApi.createTransaction = jest.fn(async () =>
-      new Promise(resolve => resolve(JUNIOR_12_MONTH_LICENCE.transActionResponse)))
+    salesApi.createTransaction = jest.fn(async () => new Promise(resolve => resolve(JUNIOR_12_MONTH_LICENCE.transActionResponse)))
```

Comment:
Why are you concatenating lines to make them too long!

Reply:
Serious point tho, I am could be concurrently working on this now and it would throw up a lot of 'merge' noise

Reply:
As mentioned in the comments, these were picked up by prettier-standard.

I'd like to understand how these aren't being formatted in the first place.   Prettier-standard should be picking these up.  The only way I think these could be getting in are if commits are being made with the --no-verify flag.

I'll look into a pre-push flag to deny the push if code is not properly formatted/linted